search-commons/src/main/java/no/unit/nva/search/models/SearchTicketsQuery.java

```diff
@@ -85,4 +92,8 @@ private void addViewingScope(ViewingScope viewingScope, BoolQueryBuilder queryBu
                 .queryName(SearchClient.EXCLUDED_VIEWING_SCOPES_QUERY_NAME);
         }
     }
+
+    private void addAggregations(SearchSourceBuilder sourceBuilder) {
+        aggregations.forEach(aggDTO -> sourceBuilder.aggregation(aggDTO.toAggregationBuilder()));
```

Comment:
We try to avoid abbreviations and use of uppercase in variable names. `aggDTO` -> `aggregation` so that is matches name of source (`aggregations`)?

Reply:
Done.